ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -1,6 +1,18 @@
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { A } from '@ember/array';
+
+class Tag {
+  @tracked key;
+  @tracked value;
+
+  constructor(key, value) {
+    this.key = key;
+    this.value = value;
+  }
+}
```

Comment:
Since the tags UI is a client-side convenience, this seems like a great solution!  Tags aren't used by any model or other code outside of this component.  There are a handful of other examples where a class is needed, and they live as close to their uses as possible.